lib/serialization/FileMiddleware.js

```diff
@@ -456,8 +455,44 @@ class FileMiddleware extends SerializerMiddleware {
 							stream.on("error", err => reject(err));
 							stream.on("finish", () => resolve());
 						}
-						for (const b of content) stream.write(b);
-						stream.end();
+						// split into chunks for WRITE_LIMIT_CHUNK size
+						const chunks = [];
```

Comment:
```
if (size < WRITE_LIMIT_CHUNK) {
   for (const b of content) stream.write(b);
   return stream.end();
}
```
I think we still can use this

Reply:
We could, but that doesn't really makes a difference...

Reply:
for small builds we will skip chunking..